searcher/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+
+	"github.com/pkg/errors"
+
+	"github.com/sleepypioneer/resource-searcher/searcher/resource"
+)
+
+// Scraper scrapes resources and stores them in the DB
+type Scraper interface {
+	Scrape(string) (*resource.Resource, error)
+	store(res *resource.Resource) error
+}
```

Comment:
There is an issue with that interface as it's not implemented by anything at the moment. Also it's not used anywhere, but it's not a blocker for this MR, we have time to fix it later on :)